spot_rl_experiments/experiments/skill_test/test_nav_pick_nav_place.py

```diff
@@ -0,0 +1,159 @@
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+# mypy: ignore-errors
+import argparse
+import atexit
+import os
+import os.path as osp
+import signal
+import sys
+import time
+from copy import deepcopy
+
+import magnum as mn
+import numpy as np
+import rospy
+from perception_and_utils.utils.generic_utils import map_user_input_to_boolean
+from spot_rl.envs.skill_manager import SpotSkillManager
+from spot_rl.utils.utils import ros_topics as rt
+from spot_wrapper.utils import get_angle_between_two_vectors
+
+NUM_REPEAT = 3
+WAYPOINT_TEST = [([5.4, 2.5], [3.6, -4.2])] * NUM_REPEAT  # x, y
+
+metrics_list = None
+metrics = None
+
+
+def print_logs():
+    print(f"Current metrics while exiting {metrics_list}, Metrics: {metrics}")
+
+
+class SpotRosSkillExecutor:
+    """This class reads the ros butter to execute skills"""
+
+    def __init__(self):
+        self.spotskillmanager = None
+        self._cur_skill_name_input = None
+
+    def compute_metrics(self, traj, target_point, name_key="nav1"):
```

Comment:
This is also repeated from the `test_dynamic_yaw_plus_mobile_gaze.py` file, can we condense the 3 files?